app/models/subscribe_link.rb

```diff
@@ -0,0 +1,180 @@
+    "overcast" => "https://overcast.fm/itunes${appleID}",
+    "pocketcasts" => "https://pca.st/itunes/${appleID}",
+    "youtube" => "https://music.youtube.com/playlist?list=${uniquePlatformID}",
+    "youtube_feed" => "https://music.youtube.com/library/podcasts?addrssfeed=${base64url$(feedURL)}",
+    "amazon" => "https://music.amazon.com/podcasts/${uniquePlatformID}",
+    "antenna" => "https://antennapod.org/deeplink/subscribe?url=${feedURL}",
+    "anytime" => "https://anytimeplayer.app/subscribe?url=${feedURL}",
+    "apollo" => "https://shows.apollopods.com/show?feedUrl=${feedURL}",
+    "breez" => "https://breez.link/p?feedURL=${encodeURIComponent(feedURL)}",
+    "castamatic" => "https://castamatic.com/guid/${podcastGUID}",
+    "castbox" => "https://castbox.fm/vic/${appleID}",
+    "castro" => "https://castro.fm/itunes/${appleID}",
+    "curiocaster" => "https://curiocaster.com/podcast/pi${podcastIndexShowID}",
+    "deezer" => "https://www.deezer.com/show/${uniquePlatformID}",
+    "fountain" => "https://fountain.fm/show/${podcastIndexShowID}",
+    "global" => "https://www.globalplayer.com/podcasts/${uniquePlatformID}",
+    "goodpods" => "https://www.goodpods.com/podcasts-aid/${appleID}",
+    "gpodder" => "https://gpodder.net/subscribe?url=${feedURL}",
+    "hark" => "https://harkaudio.com/p/${uniquePlatformID}",
+    "iheart" => "https://iheart.com/podcast/${uniquePlatformID}",
+    "lnbeats" => "https://lnbeats.com/album/${podcastGUID}",
+    "luminary" => "https://luminarypodcasts.com/listen/${slug}/${slug}/${uniquePlatformID}",
+    "moon" => "https://moon.fm/itunes/${appleID}",
+    "pandora" => "https://pandora.com/podcast/${slug}/PC:${uniquePlatformID}",
+    "player" => "https://player.fm/subscribe?id=${encodeURIComponent(feedURL)}",
+    "podbean" => "https://www.podbean.com/itunes/${appleID}",
+    "podaddict" => "https://podcastaddict.com/feed/${encodeURIComponent(feedURL)}",
+    "podapp" => "https://podcast.app/${slug}-p${uniquePlatformID}",
+    "podguru_apple" => "https://app.podcastguru.io/podcast/${appleID}",
+    "podguru_feed" => "https://app.podcastguru.io/podcast/X${hex(feedURL)}",
+    "podrepublic" => "https://www.podcastrepublic.net/podcast/${appleID}",
+    "podfriend" => "https://podfriend.com/podcast/${appleID}",
+    "podlp_apple" => "https://link.podlp.app/${appleID}",
+    "podlp_guid" => "https://link.podlp.app/${podcastGUID}",
+    "podstation" => "https://podstation.github.io/subscribe-ext/?feedURL=${feedURL}",
+    "podurama" => "https://podurama.com/podcast/${slug}-i${appleID}",
+    "podverse" => "https://api.podverse.fm/api/v1/podcast/podcastindex/${podcastIndexShowID}",
+    "snipd" => "https://share.snipd.com/show/${uniquePlatformID}",
+    "sonnet" => "https://sonnet.fm/p/${appleID}",
+    "steno" => "https://steno.fm/show/${podcastGUID}",
+    "truefans" => "https://truefans.fm/${podcastGUID}",
+    "tunein" => "https://tunein.com/podcasts/${uniquePlatformID}"
+  }
+
+  PLATFORM_ICONS = {
```

Comment:
the reason this is a thing is that some platforms have 2 options, and i thought it would be easier to map them in this hash, rather than adding a duplicate `svg`. this mapping is then used to access the proper partial in `views/icons/_#{platform}`

Reply:
that would certainly save a bunch of lines. will push up a change.